pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "nautobot-golden-config"
-version = "1.1.0"
+version = "2.0.0-beta.0"
```

Comment:
I don't think this is required, can simply go to 1.2, and update the serializer as described in the comment there.

Reply:
Ack